README.md

```diff
@@ -7,7 +9,9 @@ check out Phantom frontend repository [linked here](https://github.com/atlp-rwan
 ## Getting Started
 
 These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
 ## Pipeline
+
 - Staging app [linked here](https://phantom-codebandits-staging.herokuapp.com/) use the `develop` branch
```

Comment:
Change the links to match the deployment heroku app.
Like [https://phantom-be-codebandits-staging.herokuapp.com](https://phantom-be-codebandits-staging.herokuapp.com)